### ISSUE TITLE
[Dev] [508] Correctly label rich text editors with their respective headings

### ISSUE THREAD

**Issue by tbolt** [CMSgov/eAPD#2675]
As noted in the 508 Compliance review:
> Form field label for example “Program Introduction” should be associated with input text box so that screen reader can render the label upon receiving keyboard focus.

The "Program Introduction" section provides a TinyMCE rich-text editing input. Currently when the rich text area receives keyboard focus it announces **Rich Text Area. Press ALT-0 for help.** which does not describe what the rich text area is for. @radavis opened a ticket with TinyMCE-React to provide some support for this:
https://github.com/tinymce/tinymce-react/issues/177

### This task is done when…
 - [ ] Rich text editors have appropriate labels that are announced when keyboard focus is received
